channels/rdpdr/server/rdpdr_main.c

```diff
@@ -89,7 +137,7 @@ static int rdpdr_server_receive_client_name_request(RdpdrServerContext* context,
 	if (UnicodeFlag)
 	{
 		ConvertFromUnicode(CP_UTF8, 0, (WCHAR*) Stream_Pointer(s),
```

Comment:
Not related to your changes but we should check there's ComputerNameLen remaining bytes